interfaces/builtin/desktop_test.go

```diff
@@ -219,3 +219,42 @@ func (s *DesktopInterfaceSuite) TestStaticInfo(c *C) {
 func (s *DesktopInterfaceSuite) TestInterfaces(c *C) {
 	c.Check(builtin.Interfaces(), testutil.DeepContains, s.iface)
 }
+
+func (s *DesktopInterfaceSuite) TestDisableMountFontCache(c *C) {
+	const mockSnapYaml = `name: desktop-snap
+version: 1.0
+plugs:
+  desktop:
+    mount-font-cache: false
+`
+	plug, plugInfo := MockConnectedPlug(c, mockSnapYaml, nil, "desktop")
+	c.Check(interfaces.BeforePreparePlug(s.iface, plugInfo), IsNil)
```

Comment:
s/Check/Assert/ probably

Reply:
I think I used Check here because the tests below it are not going to panic if this one fails.  It isn't making any modifications to the plugInfo that later parts of the test depend on.